packages/core/src/codecs/netconf-codec.ts

```diff
@@ -16,14 +16,18 @@
 import { ContentCodec } from "../content-serdes";
 import { DataSchema, DataSchemaValue } from "wot-typescript-definitions";
 
+import debug from "debug";
+
+const log = debug("core/netconf-codec");
+
 /** default implementation offering JSON de-/serialisation */
 export default class NetconfCodec implements ContentCodec {
     getMediaType(): string {
         return "application/netconf";
     }
 
     bytesToValue(bytes: Buffer, schema: DataSchema, parameters: { [key: string]: string }): DataSchemaValue {
-        // console.debug(`NetconfCodec parsing '${bytes.toString()}'`);
+        // log(`NetconfCodec parsing '${bytes.toString()}'`);
```

Comment:
General question. Shall we "re-activate" those commented-out log messages

Reply:
Added in f8d1f0767fa2a0e03de18457cc33bb7ab667d825